File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/DreamwroughtBow.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Bow;

/* JADX INFO: loaded from: classes3.dex */
public class DreamwroughtBow extends Bow {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.weapon_bow_dreamwrought_bow_name;
        this.idDescription = R.string.weapon_bow_dreamwrought_bow_description;
        this.idImage = R.drawable.dreamwrought_bow;
        this.price = 1055L;
        this.dexterity = 39;
        this.intelligence = 13;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/DreamwroughtBow.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Bow

class DreamwroughtBow : Bow() {
    override fun configureProperties() {
        idName = R.string.weapon_bow_dreamwrought_bow_name
        idDescription = R.string.weapon_bow_dreamwrought_bow_description
        idImage = R.drawable.dreamwrought_bow
        price = 1055L
        dexterity = 39
        intelligence = 13
    }
}
